climada/hazard/tc_tracks.py

```diff
@@ -188,20 +190,20 @@ class TCTracks():
             - on_land (bool for each track position)
             - dist_since_lf (in km)
     """
-    def __init__(self, pool=None):
+    def __init__(self,
+                 data: Optional[List[xr.Dataset]] = None,
+                 pool: Optional[pathos.multiprocessing.ProcessPool] = None):
         """Create new (empty) TCTracks instance.
 
         Parameters
         ----------
```

Comment:
pydoc parameter description for `data` is missing

Reply:
Done: b7c6b8056647559858f28932e212e053f173bd1e